crates/typos-dict/assets/allowed.csv

```diff
@@ -6,3 +6,10 @@ deques,noun
 dequeues,verb
 ons,so `add-ons` works
 accreting,verb of accrete
+sortings,plural of sorting
```

Comment:
Can you provide a source for this?

Reply:
Merriam-Webster describes the 5th meaning of the "sorting" noun as:

> an instance of sorting

Multiple instances of sorting would thus be "sortings". See [Google Books for examples of "sortings"](https://www.google.com/search?q=%22sortings%22&tbm=bks) (currently yields 22.600 results). The English Wiktionary also has [a page](https://en.wiktionary.org/wiki/sortings) (though that probably doesn't mean much).

Reply:
Ah, right ... good catch! I'll split this off.

Reply:
I filed #752 for the further discussion on "sortings".